data/Panquesito7.json

```diff
@@ -163,6 +164,14 @@
       "color": "#0030e7",
       "description": "Grow my Minetest projects as well as help and improve the community."
     },
+    {
+      "title": "50+ followers on Dev.to",
+      "date": "March 6, 2023",
+      "icon": "FaDev",
+      "color": "#0b11bc",
+      "description": "Got 50+ followers on my Dev.to profile.",
```

Comment:
Hi @Panquesito7! 😄 This is a great start! 😄 I think you can briefly describe how you reached this milestone! :)